src/clj_meowallet/core.clj

```diff
@@ -1,11 +1,49 @@
 (ns clj-meowallet.core
-  (:require [environ.core :refer [env]]))
+  (:require [environ.core :refer [env]]
+            [clj-meowallet.http :as meowallet-http]
+            [aleph.http :as http]))
 
 (def sandbox-url "https://services.sandbox.meowallet.pt/api/v2/")
 (def production-url "https://services.wallet.pt/api/v2/")
 
-(defn url-builder
+(defn host
   []
   (cond
     (env :production) production-url
```

Comment:
Usually the env stuff will return a string, like `"true"`. Maybe we should test that. Because if we set `PRODUCTION=false` the first cond will be performed.